python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -431,13 +431,16 @@ def dragEnterEvent(self, event):
         """
         # record selection for use later.
         self._dragged_items = []
-        dragged_items = self.selectedItems()
+        dragged_items = []
 
-        # ignore any selections which aren't purely made from TopLevelTreeNodeItems
-        for item in dragged_items:
-            if not isinstance(item, TopLevelTreeNodeItem):
-                logger.debug("Selection contains non-top level nodes. Ignoring")
-                return
+        # retain only TopLevelTreeNodeItems items from selected elements
+        for item in self.selectedItems():
+            if isinstance(item, TopLevelTreeNodeItem):
+                dragged_items.append(item)
+
+        # ignore any selection that does not contain at least one TopLevelTreeNodeItems
+        if(len(dragged_items)==0):
+            return
```

Comment:
suggest adding a debug statement back in here. something like: "No top-level nodes included in selection."